WordPress/Classes/ViewRelated/Notifications/FormattableContent/WordPressKit+Notifiable.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  WordPressKit+Notifiable.swift
+//  WordPress
+//
+//  Created by Alex on 3/19/25.
+//  Copyright © 2025 WordPress. All rights reserved.
+//
```

Comment:
nit:  As far as I know, we usually don't include comments in the file header, so I believe this entire code can be removed.

Reply:
Oh yeah forgot to delete this. Thanks for the catch.